float32 x
float32 y
float32 yaw
int16 bot_num